src/video/domain.ts

```diff
@@ -251,6 +262,9 @@ export interface CreateLiveStreamParams {
   latency_mode?: LatencyMode;
   simulcast_targets?: Array<SimulcastTargetParams>;
   test?: boolean;
+  audio_only?: boolean;
```

Comment:
Won't this conflict with #170 ?

Reply:
If #170 is merged first, I don't think so?

Reply:
I'll pull again just in case